Source/WebCore/page/Navigator.cpp

```diff
@@ -233,6 +234,25 @@ void Navigator::showShareData(ExceptionOr<ShareDataWithParsedURL&> readData, Ref
     });
 }
 
+#if PLATFORM(COCOA) || ENABLE(PDFJS)
+// https://html.spec.whatwg.org/multipage/system-state.html#pdf-viewing-support
+// Section 8.9.1.6 states that if pdfViewerEnabled is true, we must return a list
+// of exactly five PDF view plugins, in a particular order.
+constexpr size_t dummyPDFPluginCount = 5;
+
+static const Vector<String>& dummyPDFPluginNames()
+{
+    static NeverDestroyed<Vector<String>> dummyPluginNames(std::initializer_list<String> {
+        String { "PDF Viewer"_s },
```

Comment:
Do we really need the `String { }`?

Reply:
I thought I did (I copied this format from elsewhere), but I'll try without.

Reply:
It seems to work properly without the String, so I will fix that.